app/views/spree/admin/webhooks_subscribers/show.html.erb

```diff
@@ -41,9 +41,9 @@
         <tr id="<%= spree_dom_id event %>">
           <td><%= event.name.html_safe %></td>
           <td><%= active_badge(event.success) %></td>
-          <td><%= event.created_at.html_safe %></td>
-          <td><%= event.response_code.html_safe %></td>
-          <td><%= event.execution_time.html_safe %></td>
+          <td><%= event.created_at %></td>
+          <td><%= event.response_code %></td>
+          <td><%= event.execution_time %></td>
           <td><%= event.request_errors.gsub('[SPREE WEBHOOKS]', '').html_safe %></td>
```

Comment:
@aleksandarpetrushev this will not sanitize the output, this will do the opposite thing - it will return it in a dangerous way